Configure Sentry in repo that uses Mikro-ORM

I tried to configure Sentry in my NestJS API, but when I installed Sentry I got this error :

> node_modules/mikro-orm/dist/utils/RequestContext.d.ts:7:11 - error TS2451: Cannot redeclare block-scoped variable 'active'.
> 
> 7     const active: ORMDomain;
>             ~~~~~~
> 
>   node_modules/@sentry/hub/dist/hub.d.ts:5:9
>     5     let active: Domain;
>               ~~~~~~
>     'active' was also declared here.

--- a/lib/utils/RequestContext.ts
+++ b/lib/utils/RequestContext.ts
@@ -1,12 +1,9 @@
-import domain from 'domain';
+import domain, { Domain } from 'domain';
 import { v4 as uuid } from 'uuid';
 import { EntityManager } from '../EntityManager';
+import { Dictionary } from '../typings';
 
-declare module 'domain' {
-  export type ORMDomain = Domain & { __mikro_orm_context?: RequestContext };
-  const active: ORMDomain;
-  function create(): ORMDomain;
-}
+export type ORMDomain = Domain & { __mikro_orm_context?: RequestContext };
 
 export class RequestContext {
 
@@ -19,7 +16,7 @@ export class RequestContext {
    */
   static create(em: EntityManager, next: (...args: any[]) => void) {
     const context = new RequestContext(em.fork(true, true));
-    const d = domain.create();
+    const d = domain.create() as ORMDomain;
     d.__mikro_orm_context = context;
     d.run(next);
   }
@@ -28,7 +25,8 @@ export class RequestContext {
    * Returns current RequestContext (if available).
    */
   static currentRequestContext(): RequestContext | undefined {
-    return domain.active ? domain.active.__mikro_orm_context : undefined;
+    const active = (domain as Dictionary).active as ORMDomain;
+    return active ? active.__mikro_orm_context : undefined;
   }
 
   /**

